.github/workflows/tm.integration.tests.yml

```diff
@@ -48,7 +48,14 @@ jobs:
         uses: actions/checkout@master
         with:
             fetch-depth: 1
+      - name: Check Go Version
+        run: echo "::set-output name=value::$(cat GO_VERSION)"
+        id: go-version
+      - name: Install Go
+        uses: actions/setup-go@v2
+        with:
+          go-version: ${{ steps.go-version.outputs.value }}
       - name: Run integration tests
         uses: ./.github/actions/tm-integration-tests
         env:
-          GOPATH: /github/workspace
+          GOPATH: /github/workspace
```

Comment:
There should be a newline at the end of `tm.integration.tests.yml`.

Reply:
Added newline.